hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java

```diff
@@ -114,6 +113,7 @@ private void initializeAndProcessMessage() {
         if (!node.joined()) {
             throw new HazelcastInstanceNotActiveException("Hazelcast instance is not ready yet!");
         }
+        parameters = decodeClientMessage(clientMessage);
```

Comment:
Why just not just call it 'decode' since the client only deals with ClientMessages.